docs/src/pages/system/spacing/spacing.md

```diff
@@ -54,7 +54,7 @@ const theme = {
 
 ```jsx
 const theme = {
-  spacing: value => value ** 2,
+  spacing: value => value * 2,
```

Comment:
```suggestion
  spacing: value => value ** 2,
```

Reply:
I saw was marked as resolved, need change here?

Reply:
The original intent was to use a power, but that's probably confusing.